tests/Resources/date.addontest.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2018-Present by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions'); // eslint-disable-line no-unused-vars
+
+describe('Date', function () {
+	it('#toLocaleString()', () => {
+		// 2020-March-1st
+		const date = new Date(Date.UTC(2020, 2, 1));
+		const options = {
+			year: 'numeric',
+			month: 'numeric',
+			day: 'numeric',
+			timeZone: 'UTC'
+		};
+
+		should(date.toLocaleString).not.be.undefined();
+		should(date.toLocaleString).be.a.Function();
+		should(date.toLocaleString()).be.a.String();
+		should(date.toLocaleString('en-US')).be.a.String();
+		should(date.toLocaleString([ 'en-US' ])).be.a.String();
+		should(date.toLocaleString([ 'en-US', 'de-DE' ])).be.a.String();
+		should(date.toLocaleString('en-US', options)).be.a.String();
+		should(date.toLocaleString([ 'en-US' ], options)).be.a.String();
+		should(date.toLocaleString([ 'en-US', 'de-DE' ], options)).be.a.String();
+		should(date.toLocaleString(options)).be.a.String();
+
+		should(date.toLocaleString('en-US', options)).be.eql('3/1/2020');
+		should(date.toLocaleString('de-DE', options)).be.eql('1.3.2020');
+		should(date.toLocaleString('ja-JP', options)).be.eql('2020/3/1');
+	});
+
+	it('#toLocaleDateString()', () => {
+		// 2020-March-1st
+		const date = new Date(Date.UTC(2020, 2, 1));
+		const options = {
+			year: 'numeric',
+			month: 'numeric',
+			day: 'numeric',
+			timeZone: 'UTC'
+		};
+
+		should(date.toLocaleDateString).not.be.undefined();
+		should(date.toLocaleDateString).be.a.Function();
+		should(date.toLocaleDateString()).be.a.String();
+		should(date.toLocaleDateString('en-US')).be.a.String();
+		should(date.toLocaleDateString([ 'en-US' ])).be.a.String();
+		should(date.toLocaleDateString([ 'en-US', 'de-DE' ])).be.a.String();
+		should(date.toLocaleDateString('en-US', options)).be.a.String();
+		should(date.toLocaleDateString([ 'en-US' ], options)).be.a.String();
+		should(date.toLocaleDateString([ 'en-US', 'de-DE' ], options)).be.a.String();
+		should(date.toLocaleDateString(options)).be.a.String();
+
+		should(date.toLocaleDateString('en-US', options)).be.eql('3/1/2020');
+		should(date.toLocaleDateString('de-DE', options)).be.eql('1.3.2020');
+		should(date.toLocaleDateString('ja-JP', options)).be.eql('2020/3/1');
+	});
+
+
```

Comment:
<!--
  1 failure:  tests/Resources/d...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/date.addontest.js;
  Line: 66;
-->
- :no_entry_sign: tests/Resources/date.addontest.js line 66 – More than 1 blank line not allowed. (no-multiple-empty-lines)